docker/dockerfile

```
@@ -1,5 +1,5 @@
 FROM node:lts-alpine3.14
 
 COPY dist /ion
-COPY json /json
+COPY json /config
```

sanity: should it be:

```suggestion
COPY config /config
```

---

yes